content/_posts/minecraft/java/2024-08-10-choosing-the-right-minecraft-server-jar.md

```diff
@@ -0,0 +1,146 @@
+
+### Arclight
+Arclight is similar to Mohist, providing a hybrid environment that supports both Forge mods and Spigot plugins. It is particularly known for its performance optimizations and compatibility with large modpacks.
+- **Performance-Oriented**: Arclight is optimized for better performance, especially when running large modpacks alongside plugins.
+- **Custom Features**: Offers some unique features and tweaks that are not available in other hybrid jars, making it a good choice for custom servers.
+
+
+<!-- Proxies: Bungee, Waterfall, Velocity -->
+
+
+## Proxies
+Proxies are server software that allow multiple Minecraft servers to be linked together, providing features like server networks, cross-server communication, and improved player management. Here are some of the most popular proxies:
+
+### BungeeCord
+BungeeCord is one of the most widely used Minecraft proxies, enabling the linking of multiple Minecraft servers into a single network. It allows players to switch between servers seamlessly without disconnecting.
+- **Ease of Use**: BungeeCord is straightforward to set up and manage, making it a popular choice for networked servers.
+- **Cross-Server Features**: Supports cross-server chat, global player lists, and more.
+
+
+### Waterfall
+Waterfall is a fork of BungeeCord that offers additional performance improvements and bug fixes. It's designed to provide a more stable and efficient proxy experience.
+- **Performance and Stability**: Waterfall improves upon BungeeCord's performance and stability, making it a better choice for larger networks.
+
+> **Note on Waterfall**: Waterfall has reached its end of life, and it is recommended to transition to Velocity. For more information, see the announcement on the [PaperMC website](https://forums.papermc.io/threads/1088/).
+
+
+### Velocity
+Velocity is the modern, high-performance proxy designed with performance and stability in mind. It’s a full alternative to Waterfall with its own plugin ecosystem.
+- **High Performance**: Velocity is known for its low-latency performance, making it ideal for large, busy networks.
+- **Enhanced Security**: It includes advanced security features, helping to protect your network from attacks and exploits.
+- **Active Development and Ecosystem**: Velocity is actively maintained and has its own plugin ecosystem, making it a forward-looking choice for modern Minecraft server networks.
+
+<!-- END -->
+
+## Conclusion
+- **For Optimal Performance**: Choose **Pufferfish** for a performance-focused server that remains close to Paper's behavior.
+- **For a Stable and Supported Server**: Opt for **Paper** if you want a balance of performance and compatibility with a wide range of plugins.
+- **For Extra Features**: Use **Purpur** if you want additional gameplay features and customization while maintaining the performance base of Pufferfish.
+- **For Basic Performance and Compatibility**: Use **Spigot** if you need a reliable server with good plugin support.
```

Comment:
Spigot doesn't support Paper plugins (duh)

Reply:
> Spigot doesn't support Paper plugins (duh)

It doesn't say anywhere that it supports Paper plugins. 99% of plugins out there are built for either Bukkit or Spigot. (duh)

Reply:
"For Basic Performance and **Compatibility**"

Reply:
my bad